scripts/sdk_installers/install_openvm_sdk.sh

```diff
@@ -43,4 +43,7 @@ else
     echo "Error: 'cargo openvm --version' failed. cargo-openvm might not have installed correctly." >&2
     echo "       Ensure ${HOME}/.cargo/bin is in your PATH for new shells." >&2
     exit 1
-fi
\ No newline at end of file
+fi
+
+# Setup aggregation keys
+cargo openvm setup
```

Comment:
Preprocess the aggregation key (not depends on the elf), which takes 5 mins on CI, and 460MB on disk space in image (I think the size should be fine).